recipes/eccodes/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
I'd start with seeing if you could run a similar cmake command on Windows. cmake seems pretty solid with different versions of VC etc. .vcproj files are usually for one version of VC, and can be fiddly to build from the command line.